.eslintrc.js

```diff
@@ -1,28 +1,27 @@
 module.exports = {
   env: {
     browser: true,
-    commonjs: true,
-    es2021: true
+    es2021: true,
   },
-  extends: ['plugin:react/recommended', 'airbnb', 'prettier', 'plugin:jsdoc/recommended'],
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  overrides: [
+  ],
   parserOptions: {
-    ecmaFeatures: {
-      jsx: true
-    },
-    ecmaVersion: 14
+    ecmaVersion: 'latest',
+    sourceType: 'module',
   },
-  plugins: ['prettier', 'jsdoc'],
+  plugins: [
+    'react',
+  ],
   rules: {
-    'react/function-component-definition': ['error', { namedComponents: 'arrow-function' }],
-    'react/jsx-props-no-spreading': 'off',
-    'react/no-unescaped-entities': 'off',
-    'react/prop-types': 'off',
-    'no-console': 'off',
-    'jsx-a11y/label-has-associated-control': ['error', { assert: 'either' }],
-    'arrow-body-style': ['error', 'as-needed'],
-    'no-param-reassign': ['error', { props: false }],
-    'jsdoc/valid-types': 'off',
-    'no-alert': 'off',
-    'import/no-extraneous-dependencies': 'off'
-  }
+    'react/prop-types': 0,
+    'react/jsx-props-no-spreading': 0,
+    'jsx-a11y/label-has-associated-control': 0,
+    'no-param-reassign': 0,
+    'no-alert': 0,
```

Comment:
Modern UX design generally prefers not to use the browser provided alert system, especially since such alerts are often used for easy dev testing. They can also lead to annoying behavior if several are fired at once. Most UX designers prefer to use application-specific modals.

However, this is an issue for UX, and given the current state of the project, it's probably best to use the built-in alert systems for now.